docs/guides/api-gateway/get-started.mdx

```diff
@@ -0,0 +1,641 @@
+Forwarding                    https://v1.api.internal -> http://localhost:8001
+
+Connections                   ttl     opn     rt1     rt5     p50     p90
+                              0       1       0.00    0.00    0.00    0.00
+
+HTTP Requests
+-------------
+
+15:00:14.035 CDT GET /cards                     200 OK
+
+```
+
+Change the header value to `2` and run the command again. You'll get the same response, but you should see this request hit the `https://v2.api.internal`
+endpoint on port `8002`.
+
+Stop both agent instances to remove the internal endpoints you just created.
+
+### Route by path
+
+Path-based routing enables you to direct traffic to different backend services based on the value of the path.
+
+#### Start internal endpoints
+
+To implement this example, run the following commands in separate terminals to start internal endpoints to the two instances of the sample API:
+
+```bash
+ngrok http 8001 --url=https://hostA.api.internal
+ngrok http 8002 --url=https://hostB.api.internal
+```
+
+While these endpoints point to two separate instances of the same backend API, they simulate the scenario where you need to direct
+traffic to two separate API services depending on the path.
+
+#### Update your policy rules
+
+This example routes traffic to two different backends depending on the path. Copy the following contents into your `policy.yaml`
+file, overwriting all previous contents:
+
+```yaml
+---
+on_http_request:
+  - name: "Route the /cards/* path to host A"
+    expressions:
+      - req.url.path.startsWith('/cards')
+    actions:
+      - type: forward-internal
+        config:
+          url: https://hostA.api.internal
+  - name: "Route the /payouts/* path to host B"
+    expressions:
+      - req.url.path.startsWith('/payouts')
+    actions:
+      - type: forward-internal
+        config:
+          url: https://hostB.api.internal
+  - actions:
+      - type: forward-internal
+        config:
+          url: https://hostB.api.internal
+          binding: internal
+```
+
+:::note
+You must always include the `forward-internal` action as the final step in your policy file since once it executes,
+no subsequent policy rules will be executed. Additionally, you always need a terminating policy rule.
+In this case, the traffic forwards to https://hostB.api.internal if the path begins with a value other than `/cards` or `/payments`.
+
+:::
+Now, run the following command to apply the new policies to your cloud endpoint.
+
+```bash
+ ngrok api endpoints update {ENDPOINT_ID} --traffic-policy "$(cat policies.yaml)"
+```
+
+#### Test traffic policies
+
+Run the following command to hit the `/cards` route:
+
+```bash
+curl -X GET https://{YOUR_NGROK_DOMAIN}/cards
+```
+
+Examine the terminal where you started the `https://hostA.api.internal` endpoint on port `8001`, and you can
+see that the request hit the correct endpoint. Repeat the steps for the `/payments` endpoint and confirm the request
```

Comment:
Saw this when helping with this rule earlier. Let's make doubly sure we're only using **endpoint** alongside cloud/internal to avoid any possible confusion.

```suggestion
see that the request hit the correct endpoint. Repeat the steps for the `/payments` route and confirm the request
```

Reply:
I saw your one comment about that and then updated `endpoints` to `routes` everywhere in the guide. Thanks for flagging that.